ocs_ci/utility/environment_check.py

```diff
@@ -184,10 +185,15 @@ def get_status_after_execution():
                 'Leftovers removed'
             ].append({f"***{kind}***": kind_diff[1]})
             leftover_detected = True
-    if leftover_detected:
-        raise exceptions.ResourceLeftoversException(
-            f"\nThere are leftovers in the environment after test case:"
-            f"\nResources added:\n{yaml.dump(leftovers['Leftovers added'])}"
-            f"\nResources "
-            f"removed:\n {yaml.dump(leftovers['Leftovers removed'])}"
-        )
+    try:
+        log.info("Checking Ceph health")
+        ceph_health_check(tries=6, delay=10)
+        log.info("Ceph cluster health is OK")
```

Comment:
ceph_health_check returns bool, hence not sure if try is needed. 
how about:

```python
if not ceph_health_check(tries=6, delay=10):
    log.info("Ceph cluster health is not OK")
```

Reply:
The try is to ensure to run the leftover_detected part.

Reply:
I think try is not going to fail here because we are not really checking the return status of ceph_health_check, so I agree try is not needed here.